packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import "@rari-capital/solmate/src/tokens/ERC721.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "./SocialContract.sol";
+
+contract Optimist is ERC721 {
+    SocialContract public immutable sc;
+    address public immutable admin;
+
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        address _admin,
+        address _sc
+    ) ERC721(_name, _symbol) {
+        admin = _admin;
+        sc = SocialContract(_sc);
+    }
+
+    function mint(address recipient) public returns (uint256) {
+        uint256 newItemId = uint256(uint160(recipient));
```

Comment:
If the token is soul bound, why not simplify this and make the recipient address the tokenId?

Reply:
I'm confused by my comment here we clearly are using the address as the id here lol ignore resolving